hub/powertoys/powerrename.md

```diff
@@ -151,28 +147,30 @@ To use the [Boost library](https://www.boost.org/doc/libs/1_74_0/libs/regex/doc/
 
 Simple matching examples
 
-| Search for       | Description                                           |
-| :---             | :---                                                  |
-| `^`              | Match the beginning of the filename (zero size)       |
-| `$`              | Match the end of the filename (zero size)             |
-| `.*`             | Match all the text in the name                        |
-| `^foo`           | Match text that begins with "foo"                     |
-| `bar$`           | Match text that ends with "bar"                       |
-| `^foo.*bar$`     | Match text that begins with "foo" and ends with "bar" |
-| `.+?(?=bar)`     | Match everything up to "bar"                          |
-| `foo[\s\S]*bar`  | Match everything between and including "foo" and "bar"|
+| Search for      | Description                                            |
+|:----------------|:-------------------------------------------------------|
+| `^`             | Match the beginning of the filename (zero size)        |
+| `$`             | Match the end of the filename (zero size)              |
+| `.*`            | Match all the text in the name                         |
+| `^foo`          | Match text that begins with "foo"                      |
+| `bar$`          | Match text that ends with "bar"                        |
+| `^foo.*bar$`    | Match text that begins with "foo" and ends with "bar"  |
+| `.+?(?=bar)`    | Match everything up to "bar"                           |
+| `foo[\s\S]*bar` | Match everything between and including "foo" and "bar" |
 
 Matching and variable examples. Capturing groups are defined in parentheses `()`. To refer to them, use `$` followed by a number: `$1` will refer to the first group, `$2` to the second etc.
 
 _When using the variables, the "Match all occurrences" option must be enabled._
 
-| Search for                 | Replace with | Description |
-| :---                       | :---         | :--- |
-| `(.*).png`                 | `foo_$1.png` | Prepends "foo\_" to the existing file name for PNG files |
-| `(.*).png`                 | `$1_foo.png` | Appends "\_foo" to the existing file name for PNG files |
-| `(.*)`                     | `$1.txt`     | Appends ".txt" extension to existing file |
-| `(^\w+\.$)¦(^\w+$)`        | `$2.txt`     | Appends ".txt" extension to existing file name only if it does not have an extension |
-| `(\d\d)-(\d\d)-(\d\d\d\d)` | `$3-$2-$1`   | Move numbers in the filename: "29-03-2020" becomes "2020-03-29" |
+| Search for                                              | Replace with | Description                                                                          |
+|:--------------------------------------------------------|:-------------|:-------------------------------------------------------------------------------------|
```

Comment:
Just a note about this table formatting: The dashes here are just a formatting preference for the markdown. Doesn't impact the way the table appears after being built.

Reply:
Hah i know, just clicked "expand" in VS Code

Reply:
Oh! Right on - haven't tried that. Good to know that's how it handles it. I've seen writers have strong opinions on what is "best" (which clearly doesn't matter... but you know, folks love to have strong opinions). 😜